.github/workflows/install-sh.yml

```diff
@@ -0,0 +1,19 @@
+name: Update install.sh Suite
+
+on:
+  push:
+    tags:
+      - v0.*
+      - v1.*
```

Comment:
Why these tags? Why not sync changes to on `master`?

Reply:
Oh, yup, missed this.